### ISSUE TITLE
All Files Disappear After Finishing The Converter??

### ISSUE THREAD

**Issue by 37media** [rrirower/DP2SceneGraphWizard#52]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Version number:**
 - Version [e.g. 1.0.0.0-alpha-8]

**Additional context**

**Comment by 37media** [comment]
So this is a doozy?!  I had been using this on one channel just fine. Thats pretty much completed so I began trying a smaller channel for it.  It created a package, that I zipped and uploaded.   But thumbnails didn't show on it.  So in an effort to fully begin again I've been starting a new package from scratch.

Only, whenever I finish thru the software, the root directory I made is empty??   I am downloading the files in Step 1, and if I take a peek thru File Explorer I do see the three folders.  Yes for some reason whenever I finish the software and go back, that entire directory & folder structure is gone?!  Almost like its being deleted out for some reason.  I've even tried fully new folders (in my Photos & Videos area to give it a fresh start), as well as deleted everything from my Recycle Bin. Does anyone have thoughts as to why this would be?   I've even restarted my computer. Again, whenever I'm done its just an empty folder with nothing in it anymore?
